### PR TITLE
Feature/scraper

### PR DESCRIPTION
**Pull request by Singletoned**
Not finished but feel free to start reviewing it @sdonk @canni

### CODE REVIEW

File: scraper/auth.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+def dictify(*args, **kwargs):
+    result = {}
+    for item in args:
+        if item:
+            result.update(item)
+    result.update(kwargs)
+    return result
+
+
+def __init__(self, login_url, username, password):
```
Comment (canni) [inline scraper/auth.py]: Huh? `__init__` on module level?

Comment (Singletoned) [inline scraper/auth.py]: Well spotted.  It was copied over from a class and I forgot to delete it.

**Comment by Singletoned** [comment]
Apparently you can reopen when you comment.

**Comment by Singletoned** [comment]
Or maybe not.

**Pull request merged**